charts/studio/values.yaml

```
@@ -68,6 +68,11 @@ global:
     # -- Celery result URL
     resultBackend: ""
 
+  # -- Studio: Settings related to DVC^X
```

```suggestion
  # -- Studio: Settings related to DVCX
```
just avoiding stuff that are hard to grep 😋